packages/browser/src/index-umd.ts

```diff
@@ -0,0 +1,16 @@
+import Client from '@bugsnag/core/client'
+// @ts-ignore
+import Event from '@bugsnag/core/event'
+// @ts-ignore
+import Session from '@bugsnag/core/session'
+// @ts-ignore
+import Breadcrumb from '@bugsnag/core/breadcrumb'
+
+import assign from '@bugsnag/core/lib/es-utils/assign'
+
+import Bugsnag from './bugsnag'
+
+export default assign(Bugsnag, { Client, Event, Session, Breadcrumb })
+export type { BrowserBugsnagStatic, BrowserConfig } from './bugsnag'
+
+export type { Client, Event, Session, Breadcrumb, Plugin } from '@bugsnag/core'
```

Comment:
Ideally we would do `export type * from '@bugsnag/core'` but that's not supported with the current version of TypeScript we're using

Reply:
TODO: Check what other types need exporting here

Reply:
Actually I don't think any types need exporting from cjs/umd entry points